pliops/consumer.cc

```diff
@@ -80,24 +85,24 @@ void Consumer::CommunicationThread()
 {
   logger->Log(Severity::INFO, FormatString("Communication thread started.\n"));
   std::unique_ptr<Connection<ConnectionType::TCP_SOCKET>> connection;
-  auto rc = Accept(*connection_, connection, timeout_msec_);
-  if (!rc.IsOk()) {
+  auto rc = Accept(*connection_, connection, connect_timeout_msec_);
```

Comment:
I think we need to differentiate accept timeout and socket r/w timeout.

Reply:
This is exactly what connect_timeout_msec_ does